### PR TITLE
[RA-2 Ch06] Spelling changes

### PR DESCRIPTION
**Pull request by wavell**
Spelling changes

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter06.md [hunk 1]
```diff
@@ -23,20 +23,20 @@
...
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which support separation of concerns during image build. A fully integrated security testing should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issues.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter06.md]: typo: "It is also important to adopt a layered packaging model which _**support**_ separation of concerns ..." ---> "... It is also important to adopt a layered packaging model which _**supports**_ separation of concerns..."

"A fully integrated security testing should be baked into the CI/CD pipeline." ---> "A fully integrated security testing _**phase (or stage)**_ should be baked into the CI/CD pipeline."

File: doc/ref_arch/kubernetes/chapters/chapter06.md [hunk 2]
```diff
@@ -89,12 +89,12 @@ Kubernetes metadata contain sensitive information including kubelet admin creden
 Logging, monitoring, alerting and log aggregation are essential for Kubernetes. Audit logs must be enabled and monitored for anomalous or unwanted API calls, especially any authorisation failure. 
 
 ##  6.11  Run-Time Security
-The following are recommnended best practices for container run-time;
+The following are recommnended best practices for container run-time:
```
Comment (ulikleber) [inline doc/ref_arch/kubernetes/chapters/chapter06.md]: There's another one: recommnended

**Comment by rabi-abdel** [comment]
@wavell would you mind please having a look at the reviews above?

File: doc/ref_arch/kubernetes/chapters/chapter06.md [hunk 3]
```diff
@@ -58,15 +58,15 @@ The following are core principles to consider when securing cloud native applica
...
+All connections to a Kubernetes cluster must be via a secure channel. The following security authentication mechanism must be adopted:
 
  - User roles and access levels must be configured to provide segregation of duties (RBAC)
  - Multi-factor authentication is mandated for all administrative access
  - Service and session authentication mechanisms must be token-based or certificate-based
  - Access control should be integrated with existing identity management platforms e.g SAML, AD, etc.
 
 ##  6.5 Use Namespaces to Establish Security Boundaries
-Namespaces in Kubernetes is the first level of isolation between components. It is easier to apply security controls (Network Policies, Pod policies, etc) to diffferent types of workloads when deployed in seperate namespaaces. 
+Namespaces in Kubernetes is the first level of isolation between components. It is easier to apply security controls (Network Policies, Pod policies, etc) to diffferent types of workloads when deployed in seperate namespaces.
```
Comment (ulikleber) [inline doc/ref_arch/kubernetes/chapters/chapter06.md:69]: One more typo: seperate ==> separate, also in next line.

Comment (rabi-abdel) [reply doc/ref_arch/kubernetes/chapters/chapter06.md]: fixed in : https://github.com/cntt-n/CNTT/pull/1375/commits/6d84c8b8b12b0346158751f8c45f66ea1e3d89d1

**Pull request merged**